src/coreComponents/common/logger/ErrorHandling.hpp

```diff
@@ -230,6 +232,30 @@ class ErrorLogger
     bool m_isValidStackTrace = false;
   };
 
+  /**
+   * @brief Retrieve all informations from the ErrorLogger and format into a stream
```

Comment:
```suggestion
   * @brief Retrieve all informations from the ErrorMsg and format into a stream.
```

And **IF** it is needed to be public, you can add:
```
   *        Useful for formating a message log text externally.
```